Src/library/store/test/src/test_basic_database.e

```diff
@@ -152,12 +152,18 @@ feature -- Spec helper
 	sql_table_name (a_name: STRING): STRING
 			-- SQL table name quoted if needed
 		local
-			l_sep: STRING
+			s: STRING
 		do
-			l_sep := db_spec.identifier_quoter
 			create Result.make_from_string (a_name)
-			Result.prepend (l_sep)
-			Result.append (l_sep)
+			if attached db_spec.identifier_quoter as l_sep then
+				if l_sep.is_valid_as_string_8 then
+					s := l_sep.to_string_8
+				else
+					s := {UTF_CONVERTER}.utf_32_string_to_utf_8_string_8 (l_sep)
```

Comment:
It looks like SQL is Unicode-capable. So, the whole implementation of the library may need to be reviewed to work with Unicode as expected: https://www.progress.com/tutorials/odbc/unicode

Reply:
Agreed, but this is not yet the purpose on this pull request.